### PR TITLE
bitopro: add watchOrders

### PR DESCRIPTION
**Pull request by sc0Vu**
Add watchOrders to monitor active orders.

### CODE REVIEW

File: ts/src/pro/bitopro.ts [hunk 1]
```diff
@@ -261,6 +261,107 @@ export default class bitopro extends bitoproRest {
...
+        //                 }
+        //             ]
+        //         }
+        //     }
+        //
+        const event = this.safeString (message, 'event');
+        const data = this.safeValue (message, 'data');
+        const symbols = Object.keys (data);
+        const limit = this.safeInteger (this.options, 'ordersLimit', 1000);
+        const orders = new ArrayCacheBySymbolById (limit);
```
Comment (carlosmiei) [inline ts/src/pro/bitopro.ts:350]: @sc0Vu should we initialize the cache every time we get an update?

Comment (carlosmiei) [reply ts/src/pro/bitopro.ts:350]: @sc0Vu can you check this comment?

Comment (sc0Vu) [reply ts/src/pro/bitopro.ts:350]: @carlosmiei they only push opened order for that topic. I still think how to cache order and update the status.
How do you think that we don't cache order for now?

Comment (pcriadoperez) [reply ts/src/pro/bitopro.ts:350]: I think keeping the cache still makes sense even if it's for open orders.

Also according to the docs they do pass the status? (https://github.com/bitoex/bitopro-offical-api-docs/blob/master/v3-1/rest-1/rest.md#order-status)

Comment (sc0Vu) [reply ts/src/pro/bitopro.ts:350]: @pcriadoperez yes, but they only return open orders. If the order is cancelled, they don't return that order.

Comment (pcriadoperez) [reply ts/src/pro/bitopro.ts]: @sc0Vu so it returns for new, partially matched and closed orders but not cancelled orders?

Comment (sc0Vu) [reply ts/src/pro/bitopro.ts]: @pcriadoperez yes

**Pull request closed without merging**